### ISSUE TITLE
The documentation achieve of past version on the website is updated incorrectly to latest version

### ISSUE THREAD

**Issue by ghost** [facebook/react-native-website#1603]
For example, In this page
https://facebook.github.io/react-native/docs/0.59/integration-with-existing-apps

The following instruction is invalid for react-native v0.59 and only works for version >=0.60
```
# The target name is most likely the name of your project.
target 'NumberTileGame' do

  # Your 'node_modules' directory is probably in the root of your project,
  # but if not, adjust the `:path` accordingly
  pod 'React', :path => '../node_modules/react-native/'
  pod 'React-Core', :path => '../node_modules/react-native/React'
  pod 'React-DevSupport', :path => '../node_modules/react-native/React'
  pod 'React-fishhook', :path => '../node_modules/react-native/Libraries/fishhook'
  pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
  pod 'React-RCTWebSocket', :path => '../node_modules/react-native/Libraries/WebSocket'

  pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
end
```

**Comment by cHaLkdusT** [comment]
I've checked this one, seems like all the previous versions has messed up 😕 

By the way, if you are using RN v0.59 below use this Podfile:

```
# The target name is most likely the name of your project.
target 'NumberTileGame' do

  # Your 'node_modules' directory is probably in the root of your project,
  # but if not, adjust the `:path` accordingly
  pod 'React', :path => '../node_modules/react-native', :subspecs => [
    'Core',
    'CxxBridge', # Include this for RN >= 0.47
    'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
    'RCTText',
    'RCTNetwork',
    'RCTWebSocket', # Needed for debugging
    'RCTAnimation', # Needed for FlatList and animations running on native UI thread
    # Add any other subspecs you want to use in your project
  ]
  # Explicitly include Yoga if you are using RN >= 0.42.0
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  # Third party deps podspec link
  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'

end
```

**Comment by chrisbobbe** [comment]
Mm, it looks like several pages, including this one, have been selected as not being handled per-version — the most up-to-date doc shows no matter which version you've selected. I reported this confusion at https://github.com/facebook/react-native-website/issues/33#issuecomment-601453102.

**Comment by chrisbobbe** [comment]
Just to add to @cHaLkdusT's helpful comment that that example Podfile doesn't come out of the blue; that's what this doc correctly listed, for v0.59 — but then it was updated for v0.60 (see https://github.com/facebook/react-native-website/commit/483eafb494298609e2865bdfaf64e6624ff2c4fc), and v0.59 users were left with an incorrect example. 🙂

**Comment by chrisbobbe** [comment]
Actually, this confusion may be resolved in 53cd8f6f3a2fa6799f9cf70c2de80e9e58e7a0f6, which adds this helpful line:

```diff
-The `Podfile` will contain a boilerplate setup that you will tweak for your integration purposes. In the end, `Podfile` should look something similar to this:
+The `Podfile` will contain a boilerplate setup that you will tweak for your integration purposes.
+
+> The `Podfile` version changes depending on your version of `react-native`. Refer to https://react-native-community.github.io/upgrade-helper/ for the specific version of `Podfile` you should be using.
+
+Ultimately, your `Podfile` should look something similar to this:
```

It looks like the link to https://react-native-community.github.io/upgrade-helper/ was added with the knowledge that only the latest version of this file would be presented, for all versions. It alerts the reader to the fact that the proper Podfile might be different than the one displayed, and gives them a way to find the correct one. Obviously, for this to be effective, the change in 53cd8f6f3a2fa6799f9cf70c2de80e9e58e7a0f6 will have to be published. 🙂

**Comment by Simek** [comment]
Hi @ghost, I'm sorry to hear that you are experiencing issue with the docs for older RN release, but the `0.59` version and versions below have been moved to the React Native Archive and pages in there won't be actively maintained anymore (but it also means that the won't be updated by a mistake on the new release). If fixing this issue is important for you feel free to send the PR with a fix targeting `archive` branch in this repository.
